### ISSUE TITLE
ssc build --platform=android (v0.3.1 not working in ubuntu)

### ISSUE THREAD

**Issue by gowram** [socketsupply/socket#475]
When i try to run "ssc build --platform=android" with v0.3.1  it's not working.
whereas **v0.2.13 it was working fine** without any issues. 

### OS - Ubuntu 22.04
![Web capture_31-5-2023_175419_](https://github.com/socketsupply/socket/assets/551819/5a2d6a86-f197-42f9-adb8-a2e2ecd5072b)

### What did you expect to see and what you saw instead?
it should complete build.

**Comment by mribbons** [comment]
Hi @gowram,
Thanks for reporting.
Can we check a few details please:
1. How did you install socket?
2. `ssc env`
3. `VERBOSE=1 ssc build -r -o --platform=android`
 
This should provide a bit more detail on why gradle is failing.

**Comment by gowram** [comment]
Hi mribbons,

I just installed using npm install, after that i did ssc setup
I did same process for v0.2.13 and v0.3.1. It is working when i switch back **v0.2.13** whereas not working for **v0.3.1**.

![image](https://github.com/socketsupply/socket/assets/551819/3eb163c6-df15-43dd-b7ae-fdf59d28fb59)

**Comment by mribbons** [comment]
Thanks @gowram, I see the problem.

We'll get the fix into 0.3.3 as 0.3.2 has already started shipping.

**Comment by mribbons** [comment]
Sorry @gowram, I was able to repro your issue by mangling gradlew. Would you mind providing some more details? Please run the following, and paste the output:
```bash
find . -name "gradlew" | while read path; do ls -l "$path"; head -n1 "$path"; done
sh -c "uname -a"
ls -l $(which sh)
```

:pray:

**Comment by gowram** [comment]
These are the outcome...
![image](https://github.com/socketsupply/socket/assets/551819/defce0da-bb23-4c7b-877e-dd6353e7942b)

**Comment by mribbons** [comment]
Thanks!

Would you mind changing the issue to Ubuntu 22.04? We are still working on our 20.04 build.

I'll keep investigating.

**Comment by gowram** [comment]
Done

**Comment by mribbons** [comment]
Hi @gowram,
I have some more commands to try. I would like to compare your output for the two different versions.

```bash
npm rm -g @socketsupply/socket
npm i -g @socketsupply/socket@0.2.13
yes | ssc setup --platform=android
ssc -v # post the version data for us to review
rm -rf build
VERBOSE=1 strace -f -o trace0.2.13.log ssc build -o --platform=android

npm rm -g @socketsupply/socket
npm i -g @socketsupply/socket@0.3.1
yes | ssc setup --platform=android
ssc -v # post the version data for us to review
VERBOSE=1 strace -f -o trace0.3.1.log ssc build -o --platform=android

zip trace.zip trace0.2.13.log trace0.3.1.log
rm trace0.2.13.log trace0.3.1.log
```

Then if you could attach trace.zip, that would be awesome!

**Comment by gowram** [comment]
Hi @mribbons 

Zip file has more than 50mb file, unable to attach it here... 
I uploaded here...
https://filebin.net/zc3tgu1fr5qhobec

**Comment by mribbons** [comment]
Excellent @gowram, thanks for sticking with it, I'm not sure why the 0.2.13 log was so big.

Please post the version info from 0.3.1:
```bash
ssc -v
```

**Comment by mribbons** [comment]
FYI @gowram, have figured it out.

If you don't need `script = ` in socket.ini, commenting it out will resolve the issue for now.

`0.3.3` will release late next week.

Thanks again for providing the necessary info to debug.

**Comment by mribbons** [comment]
Note that this fix has now been published to npm with  v0.3.3.
